core/src/main/java/io/seata/core/exception/AbstractExceptionHandler.java

```diff
@@ -144,6 +144,9 @@ public <T extends AbstractTransactionRequest, S extends AbstractTransactionRespo
         } catch (RuntimeException rex) {
             LOGGER.error("Catch RuntimeException while do RPC, request: {}", request, rex);
             callback.onException(request, response, rex);
+        } catch (Exception ex) {
```

Comment:
Can `callback.execute` throw other exceptions besides `RuntimeException` and `TransactionException`?

Reply:
SQLException in DruidAbstractDataSource